src/lib.rs

```diff
@@ -28,8 +28,12 @@
 //!
 //! The basic use of the log crate is through the five logging macros: [`error!`],
 //! [`warn!`], [`info!`], [`debug!`] and [`trace!`]
-//! where `error!` represents the highest-priority log level, and `trace!` the lowest.
-//!
+//! where `error!` represents the highest-priority log messages
```

Comment:
Maybe we could simplify this explanation a bit to something like:

_where `error!` represents the highest-priority log messages and `trace!` the lowest. The log messages are filtered by configuring the log level to exclude messages with a lower priority._

What do you think?